<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string>C:\Users\simon\Desktop\top_level_bitmap.bin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="load_filebutton">
          <property name="text">
           <string>load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="flash_button">
          <property name="text">
           <string>FLASH</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="com_name">
          <property name="text">
           <string>COM5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connectUart_Button">
          <property name="text">
           <string>Connect UART</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="test_connection_button">
        <property name="text">
         <string>TestBoard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="read_flash_button">
        <property name="text">
         <string>read Flash</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="output"/>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>476</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
